src/uneditableTextField/index.tsx

```diff
@@ -43,7 +48,7 @@ export const UneditableTextField = ({
         {children}
       </div>
       <style jsx>{style}</style>
-    </Component>
+    </Fragment>,
```

Comment:
Why fragment?

Reply:
We cannot add several components at once, actually `React.createElement` is expecting only one child